src/tests/query-temporal-evolution-by-type-and-attrs.js

```diff
@@ -0,0 +1,82 @@
+import { SharedArray } from 'k6/data';
+import { createEntity } from '../api/create-entity.js';
+import { updateAttributes } from '../api/update-attributes.js'
+import { queryTemporalEvolution } from '../api/query-temporal-evolution.js'
+import { uuidv4 } from 'https://jslib.k6.io/k6-utils/1.4.0/index.js';
+import { randomItem } from 'https://jslib.k6.io/k6-utils/1.2.0/index.js';
+
+export const options = {
+    vus: 10,
+    iterations: 100000,
+    thresholds: {
+        http_req_failed: ['rate<0.01'],
+        query_temporal_evolution: ['avg<200', 'p(95)<400']
+    },
+    setupTimeout: '5m'
+};
+
+const observedProperties = ['ammonium', 'waterTemperature', 'dissolvedOxygen'];
+
+const entities = new SharedArray('template entity', function () {
+    return JSON.parse(open('../data/template_entity.json')).entities;
+});
+
+const attributesFragment = new SharedArray('update attributes fragment', function () {
+    return JSON.parse(open('../data/update_attributes_fragment.json')).attributes;
+});
+
+export function generateRandomAttributesFragment() {
+    var now = new Date();
+    const attributes = Object.assign({}, attributesFragment[0]);
+
+    const updatedAmmonium1 = Object.assign({}, attributes.ammonium[0], {
+        value: (Math.random() * 100),
```

Comment:
what about having the same value for each attribute?

Reply:
actually, I'm wondering how it can impact the results (if you spend 3ms preparing a query that takes 20ms to respond...). so better be performant in the preparation.

Reply:
For this test I don't think this is a big impact since it is done in the setup.  But it may have more impact when testing updates attributes.